test/resolve/forceIgnore.test.ts

```diff
@@ -204,41 +98,4 @@ describe('ForceIgnore', () => {
       expect(forceIgnore.denies(manifestPath)).to.be.true;
     });
   });
-
-  describe('Defaults with old parser', () => {
-    let forceIgnore: ForceIgnore;
-    const root = join('some', 'path');
-
-    beforeEach(() => {
-      env.stub(fs, 'readFileSync').returns('');
-      forceIgnore = new ForceIgnore();
-    });
-
-    it('Should ignore files starting with a dot', () => {
-      const dotPath = join(root, '.xyz');
-      expect(forceIgnore.accepts(dotPath)).to.be.true;
-      expect(forceIgnore.denies(dotPath)).to.be.false;
-    });
-
-    it('Should ignore files ending in .dup', () => {
-      const dupPath = join(root, 'abc.dup');
-      const telemetrySpy = env.spy(Lifecycle.prototype, 'emit');
-      expect(telemetrySpy.called).to.be.false;
-
-      expect(forceIgnore.accepts(dupPath)).to.be.true;
-      expect(forceIgnore.denies(dupPath)).to.be.false;
-    });
-
-    it('Should ignore files named package2-descriptor.json', () => {
-      const descriptorPath = join(root, 'package2-descriptor.json');
-      expect(forceIgnore.accepts(descriptorPath)).to.be.true;
-      expect(forceIgnore.denies(descriptorPath)).to.be.false;
-    });
-
-    it('Should ignore files named package2-manifest.json', () => {
```

Comment:
These should stay because they're testing the default entries

Reply:
@WillieRuemmele these were only testing the old parser, the same tests are still present: https://github.com/forcedotcom/source-deploy-retrieve/blob/dee6f33901df78b886f2c7e06480ce4618d33826/test/resolve/forceIgnore.test.ts#L66-L98